scripts/merge-pr.sh

```diff
@@ -104,6 +104,13 @@ check_is_pr_single_commit() {
   fi
 }
 
+request_to_squash() {
+  if [[ $(git rev-list $BRANCH..$PR_LOCAL_BRANCH | wc -l) -gt 1 ]] ;then
+    confirm "PR has multiple commits, do interactive rebase?"
+    git rebase -i $BRANCH
```

Comment:
A simpler way would be something like:
```bash
MSG=$(git log --format=%B -n 1 $PR_LOCAL_BRANC)
git reset --soft $BRANCH
git commit -m "$MSG"
```

Reply:
Hm, no, but that would take last commit message, not first commit message...

Reply:
my thought process was that having an interactive rebase is better than nothing :-D later we can improve the flow even more

Reply:
That's fair.

Reply:
Though maybe a message or a link to how to use `rebase -i` would be nice.